include/helper_functions.php

```diff
@@ -1,15 +1,169 @@
 <?php
 
+// --------------------------------------------------------------------------
+
 function debugOutput($input){
     echo "<pre>";
     var_dump($input);
     echo "</pre>";
 }
 
+// --------------------------------------------------------------------------
+
+function emptySpace(){
+    echo"
+    <br></br>
+    ";
+}
+
+// --------------------------------------------------------------------------
+
 function formatDate($date, $format) {
     $newDateFormat = $date;
     $newDateFormat = new DateTime($newDateFormat);
     $newDateFormat = $newDateFormat->format($format);
     return $newDateFormat;
 }
 
+// --------------------------------------------------------------------------
+
+function calculateStudentTable($allStudents){
+    
+    global $remainingCredits, $classesBookedAmount, $daysToNextCLass;
```

Comment:
I was testing out how global variables work. Now, what about passing variables by reference? In C++, is very common to pass by reference. This is what faculty tells us, not necessarily real-life experience. The reason behind this is that if your program has pretty heavy data structures (thousands students stored in a linked list for instance) you should pass by reference to avoid making a copy in memory of that data when passing it to a function and work with the original value instead. Is this a thing in Web development too? (I also asked Tyler)